test_unstructured/partition/test_auto.py

```diff
@@ -385,6 +385,12 @@ def test_auto_partition_formats_languages_for_tesseract():
         )
 
 
+def test_auto_partition_element_metadata_user_provided_languages():
+    filename = "example-docs/chevron-page.pdf"
+    elements = partition(filename=filename, strategy="ocr_only", languages=["eng"])
+    assert elements[0].metadata.languages == ["eng"]
```

Comment:
maybe also add a test for no input and assert the `languages` attribute is `None`?

Reply:
if the language parameter is not passed to partition_text (the only function that implements language detection so far), the default value will be auto -> the resulting language will be detected, not `None`. still good to test for no input but the result will be an actual language list. the change has not yet been extended to other partition functions and auto

Reply:
wait, @newelh I think @shreyanid is saying we should add a test without input for `languages` and test the output. And I think that is important to test with and without input for the kwarg. The rest of this PR looks good.

Reply:
Yep I added a test here: https://github.com/Unstructured-IO/unstructured/commit/d3bfa02d143fe6412a994bfeb606eb815da2c67f

Reply:
But I can also add one with the missing kwarg for auto

Reply:
auto partition does not yet have language detection, so this partition_text test looks good for now! thank you